library/src/c_api/mod.rs

```diff
@@ -212,6 +237,22 @@ pub unsafe extern "C" fn shorebird_free_string(c_string: *mut c_char) {
     }
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn shorebird_free_update_result(result: *mut UpdateResult) {
+    if result.is_null() {
+        return;
+    }
+    unsafe {
+        // drop the message if it's not null
```

Comment:
This is good, but you can also just call shorebird_free_c_string(result.message) and save yourself some typing.